docs/Contribute/ContributeWrite/WriteSoftware.md

```diff
@@ -4,6 +4,11 @@
 If you have trouble finding a string or contributing to the software stack, ask the [community](../../Community.md)
 :::
 
+
+::: tip
+If your text change is significant, contextual change [open an issue on GitHub](https://github.com/btcpayserver/btcpayserver/issues/new/choose) and explain what you'd like to change and why.:::
```

Comment:
```suggestion
If your text change is significant or a contextual change, [open an issue on GitHub](https://github.com/btcpayserver/btcpayserver/issues/new/choose) and explain what you'd like to change and why.
:::
```

Reply:
That's not the context I want here. If I change is significant, contextual change is exactly what my goal is , what's not clear here?

Reply:
Oh right, you mean that it's significant **and** contextual.
I understood it as **or**, discard !

Reply:
Yeah it's unclear, will just put significant

Reply:
Sorry, didn't get a popup for that comment.
ACK.